pages/_app.tsx

```diff
@@ -115,11 +116,13 @@ function App({ envConfig, ...appProps }: CustomAppProps) {
       <ConfigProvider envConfig={envConfig}>
         <AppWagmiConfig>
           <AppProviderWeb3>
-            <VotePromptProvider>
-              <ModalProvider>
-                <AppRootMemo {...appProps} />
-              </ModalProvider>
-            </VotePromptProvider>
+            <VoteFormActionsProvider>
```

Comment:
I would like it more to not expose such a heavy feature-loaded provider to all the pages of the app while it will be used by only one page (if I get it right), preferring to use it only at the specific pages where it's features is being used.

Reply:
Fixed in 9f64600e8cd6f229383265d0f488ab979d804b0e